class ConnectError(Exception):
    pass


class ForwardError(Exception):
    pass
